WellCoffeeInventoryAPI/src/main/java/com/team_lightyear/WellCoffeeInventoryAPI/controllers/AccountController.java

```diff
@@ -1,4 +1,39 @@
 package com.team_lightyear.WellCoffeeInventoryAPI.controllers;
 
+import com.team_lightyear.WellCoffeeInventoryAPI.models.Account;
+import com.team_lightyear.WellCoffeeInventoryAPI.repositories.AccountRepository;
+import com.team_lightyear.WellCoffeeInventoryAPI.services.AccountService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/**
+ * Created by Dominique Gould
+ */
+
+
+@RestController
+@RequestMapping("/api/manage")
 public class AccountController {
+    @Autowired
+    private AccountRepository accountRepository;
+
+    @Autowired
+    private AccountService accountService;
+
+    @GetMapping("")
+    public List<Account> getAllAccounts() {
+        return accountService.getAllAccounts();
+    }
+
+
+    }
+
+
+
+
+
 }
```

Comment:
This closing bracket is an extra.  IntelliJ won't run the code with it here.  It will need to be deleted.

Reply:
Resolved